static/src/javascripts/projects/common/modules/commercial/user-features.js

```diff
@@ -31,6 +31,7 @@ const forcedAdFreeMode: boolean = !!window.location.hash.match(
 
 const userHasData = (): boolean => {
```

Comment:
i personally wonder whether we should be checking for the presence of the cookies at all, rather than just deleting them every time anyway

Reply:
I believe the intention here is to assume that the user is entitled to various things, rather than assume not as would be the case if we always kill cookies, on the grounds that we may not receive the response with the appropriate permissions in time, every time.

Reply:
fair enough, thanks for the background